libs/sdk-ui-dashboard/src/_staging/drills/test/InsightDrillDefinitionUtils.test.ts

```diff
@@ -8,7 +8,6 @@ import {
 import { idRef, uriRef } from "@gooddata/sdk-model";
 import { IAvailableDrillTargets } from "@gooddata/sdk-ui";
 import cloneDeep from "lodash/cloneDeep";
-import { newInsightMap, newMapForObjectWithIdentity } from "../../metadata/objRefMap";
 import {
```

Comment:
staging code using mocks from dashboard model.. this is going to kick us three steps ahead